AppTests/App/Logic/AnalyticsLogicTests.swift

```diff
@@ -226,7 +221,8 @@ extension AnalyticsLogicTests {
     let outcome = ExposureDetectionOutcome.noDetectionNecessary
 
     try Logic.Analytics.SendOperationalInfoIfNeeded(outcome: outcome).sideEffect(context)
-    XCTAssertEqual(dispatchInterceptor.dispatchedItems.count, 0)
+    try XCTAssertNotContainsType(dispatchInterceptor.dispatchedItems, Logic.Analytics.StochasticallySendOperationalInfoWithExposure.self)
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: AppTests/App/Logic/AnalyticsLogicTests.swift;
  Line: 224;
-->

- :warning: swiftlint (0.39.2): Line should be 130 characters or less: currently 137 characters (line_length)